rosco-web/pkg_scripts/postInstall.sh

```diff
@@ -27,13 +35,18 @@ remove_temp_dir() {
 }
 
 install_packer() {
-  PACKER_VERSION="1.8.1"
-  local packer_version=$(/usr/bin/packer --version)
+  PACKER_VERSION="1.10.1"
+  local packer_version="$(/usr/bin/packer --version)"
   local packer_status=$?
   if [ $packer_status -ne 0 ] || [ "$packer_version" != "$PACKER_VERSION" ]; then
-    wget https://releases.hashicorp.com/packer/${PACKER_VERSION}/packer_${PACKER_VERSION}_linux_amd64.zip
+    wget https://releases.hashicorp.com/packer/${PACKER_VERSION}/packer_${PACKER_VERSION}_linux_${ARCH}.zip
     unzip -o "packer_${PACKER_VERSION}_linux_amd64.zip" -d /usr/bin
```

Comment:
need to adjust this as well to use $ARCH

Reply:
🤦‍♂️

Reply:
Thanks! Fixed 👍